requirements.txt

```diff
@@ -2,3 +2,4 @@ pbr>=1.3
 fixtures
 python-subunit >= 0.0.18
 testtools >= 0.9.30
+PyYAML>=3.1.0
```

Comment:
So JSON would be possible, but it's definitely much more difficult to read and write as a human, which is the use case here. Using an ini file would be awkward here because of the data structure being used. It doesn't really have the semantics to describe that easily. I don't think adding yaml is that much of an overhead weighed against the ui in this case.